solr/CHANGES.txt

```diff
@@ -73,6 +73,8 @@ New Features
 ---------------------
 * SOLR-14496: Solr CLI commands now can interact with a Solr secured using Basic Authentication.  (Eric Pugh)
 
+* SOLR-17006: User-defined core properties are persisted (Vincent Primault)
```

Comment:
Add this in 9.5.0 (hoping this can be backported if deemed worthwhile and innocuous enough).